modules/podman.js

```diff
@@ -121,18 +128,21 @@ class Container {
 
 /** discoverPodmanVersion fetches the podman version from cli */
 // eslint-disable-next-line no-unused-vars
-function discoverPodmanVersion() {
-    const [res, out, err, status] = GLib.spawn_command_line_sync("podman version --format json");
-    if (!res) {
-        Logger.info(`status: ${status}, error: ${err}`);
+async function discoverPodmanVersion() {
+    let versionJson;
+
+    try {
+        const out = await spawnCommandline("podman version --format json");
+        versionJson = JSON.parse(out);
+    } catch (e) {
+        Logger.info(e.message);
         throw new Error("Error getting podman version");
     }
-    Logger.debug(out);
-    const versionJson = JSON.parse(imports.byteArray.toString(out));
-    if (versionJson.Client !== null && versionJson.Client.Version !== null) {
-        podmanVersion = new Version(versionJson.Client.Version);
-    }
-    if (versionJson === null) {
+
+    const versionString = versionJson?.Client?.Version;
```

Comment:
nice, I didn't know javascript has that feature. Too bad gnome-3.32.2 runs gjs 1.56. Not a big issue for porting though

Reply:
Right, I can change that to something like
```js
    const versionString = versionJson && versionJson.Client
        ? versionJson.Client.Version : undefined;
```
if you prefer.